dotcom-rendering/src/client/poorPerformanceMonitoring.ts

```diff
@@ -43,20 +43,27 @@ const getTimeToFirstByte = async () =>
 		}
 	}));
 
-export const performanceMonitoring = async (): Promise<void> => {
-	try {
-		const [fcp, ttfb] = await Promise.all([
-			getFirstContentfulPaint(),
-			getTimeToFirstByte(),
-		]);
+/** Whether or not the current page is running more slowly than acceptable */
+export const isPerformingPoorly = Promise.all([
+	getFirstContentfulPaint(),
+	getTimeToFirstByte(),
+])
+	.then(([fcp, ttfb]) => ttfb > TTFB_THRESHOLD && fcp > FCP_THRESHOLD)
+	.catch(() => true);
```

Comment:
Browsers that [do not support `PerformanceObserver`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver#browser_compatibility), which is required to get FCP are:

- Chrome < 52
- Edge < 79
- Firefox < 57
- Opera < 39
- Safari < 11